smap/sources/conf/twitter_conf.ini

```diff
@@ -13,4 +13,4 @@ consumer_secret = XXXXXXXXXXXXXXXXXXXXXX
 access_token_key = XXXXXXXXXXXXXXXXXXXXXX
 access_token_secret = XXXXXXXXXXXXXXXXXXXXX
 Timezone = Asia/Kolkata
-url = http://52.20.89.53/
```

Comment:
Yes. The twitter driver runs on server, and hence for the server the url is localhost.